servicex_app/pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[tool.poetry]
+name = "servicex-app"
+version = "1.0.0-RC.3"
+homepage = "https://iris-hep.org"
+license = "BSD"
+maintainers = ["ServiceX Team <bengal1@illinois.edu>"]
+authors = []
+description = "REST Frontend to ServiceX"
+readme = "README.rst"
+packages = [{include = "servicex_app"}]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+MarkupSafe = "^2.1.1"
+Jinja2 = "^3.1.2"
+Flask = "^2.2.2"
+Flask-WTF = "^1.0.1"
+Flask-Cors = "^3.0.10"
+WTForms = "^3.0.1"
+email-validator = "^1.3.0"
+pika = "^1.3.0"
+Flask-RESTful = "^0.3.9"
+Flask-JWT-Extended = "^4.4.4"
+passlib = "^1.7.4"
+flask-sqlalchemy = "^3.0.1"
+kubernetes = "^24.2.0"
+globus-sdk = "^3.12.0"
+cryptography = "^38.0.2"
+Bootstrap-Flask = "^2.1.0"
+humanize = "^4.4.0"
+python-logstash = "^0.4.8"
+gunicorn = "^20.1.0"
+pytest = "^7.1.3"
```

Comment:
These should be in a test group and not bundled in with the production deploy

Reply:
test groups have been created for each repository